stl/inc/chrono

```diff
@@ -5543,10 +5640,12 @@ struct _Chrono_formatter {
                     continue;
                 }
 
-                // TRANSITION, out of bounds may be legal for non-localized decimal printing, but it isn't very
-                // consistent.
-                if (!_Val.ok()) {
-                    _THROW(format_error("Cannot print out-of-bounds time point."));
+                if (_Needs_localization(_Spec)) {
+                    if (!_Val.ok()) {

```

Comment:
This is handling a similar case, but narrows the scope to only localized specs. Since localization is defined by `time_put`, and that does not handle `!ok()` values, we are always safe to throw. The comment refers to non-localized decimal printing, which this PR takes care of.